src/index.js

```diff
@@ -65,13 +70,58 @@ export default class TagInput extends React.Component {
     }
   }
 
+  fetchServer(query) {
+    const { fetchUrl, server } = this.props;
+
+    this.setState({
+      waiting: true
+    });
+
+    return new Promise((resolve, reject) => {
+      superagent
+        .get(fetchUrl)
+        .set('x-jira-server', server)
```

Comment:
Instead of hard coding this, let's pass a headers object in so in the future we can use it for other things. I'm thinking we might need to pass around JWTs and not all requests are going to need `x-jira-server`

```
let req = superagent.get(fetchUrl);

Object.keys(headers).forEach((key)=>{
 res.set(key, headers[key]);
});

req.end(...
```